packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -55,7 +55,7 @@ function toTimestamp(date) {
   }
 
   const ms = parseDateTime(date)
-  if (!ms) {
+  if (!ms || ms.getTime() === 0) {
```

Comment:
`ms` should contain milliseconds. So maybe something like that:
```js
const ms = parseDateTime(date)?.getTime?.()
if (ms === undefined || ms === 0) {
  return null
}

return Math.round(ms / 1000)
```

Reply:
I think the second `?.` is unnecessary.

Also you can use the ternary operator instead of the `if` statement.